### PR TITLE
Dark redesign

### PR DESCRIPTION
**Pull request by robalb**
Changes:
- dank redesign
- new pages:
  Leaderboard page
  User page
- splitted js and css in modules, shared by the different pages
- reduced the amount of api calls from index: now it doesn't update when the tab is minimized or out of focus

 Todo
Improve design for the user page, and add other features (show active / old investments, chart of the activities, user badges)

### CODE REVIEW

**Comment by robalb** [comment]
Note: this fixes #157 and part of #146 .
I have tested the website only on firefox for windows
And on chrome for android.
Explorer is not supported

File: docs/wall.html [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+    <link type="text/css" rel="stylesheet" href="css/wall.css"/>
+    <script type="module" src="js/wall.js"></script>
+
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+</head>
+
+<body>
+    <nav>
+        <div class="nav-wrapper">
+            <a href="#" class="brand-logo"><img src="resources/logo240.png" class="logo-img"/><span
```
Comment (rickles42) [inline docs/wall.html]: Let's have the nav logo and name bring the user back to index.html

File: docs/wall.html [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+    
+    <div id="announcements-modal" class="modal ">
+    </div>
+
+<!-- PUT PAGE CODE HERE -->
+
+    <div class="container" >
+      <div class="section center-align">
+        <h4 class="no-margin-top" >Top Traders wall</h4>
+        <p>Select one of the market seasons below to view it's top 50 traders.</p>
```
Comment (rickles42) [inline docs/wall.html]: The purpose of this page is already clear. We can simplify the instructions to "Select a season" or even remove them entirely and just put a label next to the buttons that says, "Season:"

File: docs/wall.html [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+    </ul>
+    
+    <div id="announcements-modal" class="modal ">
+    </div>
+
+<!-- PUT PAGE CODE HERE -->
+
+    <div class="container" >
+      <div class="section center-align">
+        <h4 class="no-margin-top" >Top Traders wall</h4>
```
Comment (rickles42) [inline docs/wall.html]: Let's be consistent in the naming of this page, and always call it "Leaderboards"

File: docs/wall.html [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+         <h5>menu</h5>
+         <a class="sidenav-close" href="#!"><i class="material-icons">close</i></a>
+        </li>
+        <li><a id="without-this-everything-brokes">lol</a></li>
+        <li class="sidebar-search">
+            <input id="investor-username-mobile" placeholder="search user" type="search"  >
+        </li>
+        <!-- <li><a href="foo.html">brokers</a></li> -->
+        <li><a href="index.html">home</a></li>
+        <li><a class="grey-text" >leaderboard</a></li>
```
Comment (rickles42) [inline docs/wall.html]: Change nav to "leaderboards"

File: docs/index.html [hunk 5]
```diff
@@ -158,7 +165,10 @@ <h4 class="row-title">Overview</h4>
                     <div class="col s12 m4 l4">
                         <h4>Top Investors</h4>
                         <p>This is a list of the top 5 investors since 2018-06-29<br>
+                        <a href="./wall.html" class="underlined" >view the wall of fame</a>
```
Comment (rickles42) [inline docs/index.html]: Change to something like "view full leaderboards"

File: docs/index.html [hunk 6]
```diff
@@ -82,14 +89,14 @@
...
                     class="material-icons left">show_chart</i>learn more</a>
             </p>
             <div class="divider"></div>
             <div class="section">
                 <div class="row">
                     <div class="col s12 m4 l3">
-                        <h4 class="row-title">Overview</h4>
+                        <h4 class="no-margin-top">Overview</h4>
                         <p>Market active since 2018-06-29</p>
                         <p class="overview-text">Active investments: <span id="active-investments">--</span>, detaining
```
Comment (rickles42) [inline docs/index.html]: Let's clean up the summary text to three lines:

"Active investments: ---"
"Total invested: --- M¢"
"Total balances: --- M¢"

File: docs/wall.html [hunk 7]
```diff
@@ -0,0 +1,225 @@
...
+          <h5 id="right-name">-</h5>
+          <p><span id="right-net-worth">0</span> <span id="right-net-worth-suffix"></span><br>
+          <a id="right-profile">profile</a></p> 
+        </div>
+        
+
+        
+      </div>
+      
+      <div class="table-wrap" >
```
Comment (rickles42) [inline docs/wall.html]: Can we clean up the sizing of the leaderboards columns? On my laptop (using Edge browser) there's a bunch of empty space to the right of "Completed investments". Looks like the table should be about 75% of its current width.

**Review by rickles42 — CHANGES_REQUESTED** [review]
Looks great overall, just have some minor feedback

**Comment by rickles42** [comment]
Also, I got a "INFO: Write your message here" popup when I first launched the page - I feel like we should just remove that bit until we want to push another message"

File: docs/wall.html (hunk 7 above)
Comment (robalb) [reply docs/wall.html]: Can you send me a screenshot? Just to see if the problem is related to edge, or your screen dimensions

Comment (rickles42) [reply docs/wall.html]: @robalb Sent to you on Slack.

**Comment by robalb** [comment]
ok, i've 
- fixed the logo that didn't link to the index
- cleaned the summary text
- fixed the naming inconsistency for the leaderboards page
- disabled the info popup

the leaderboards table width is a little bit more complicated to fix, i'll do this another time, i also have to add a limit to how much the other pages can stretch horizontally.

**Review by rickles42 — APPROVED** [review]

**Pull request merged**